tests/transformers/helper.py

```diff
@@ -287,6 +287,20 @@ def load_peft_pipeline():
         )
 
 
+@prefetch
+@flaky()
+def load_custom_code_pipeline():
+    model = transformers.AutoModel.from_pretrained(
+        "hf-internal-testing/test_dynamic_model_with_util", trust_remote_code=True
```

Comment:
lots of dummy models in hf-internal-testing: https://huggingface.co/hf-internal-testing

looks like this one specifically was made for testing custom code models with relative imports: https://huggingface.co/hf-internal-testing/test_dynamic_model_with_util/tree/main

Reply:
beautiful! These are ideal!! Great find @daniellok-db